src/js/disability-benefits/containers/ClosedClaimsPage.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getClaims, getFilteredClaims, changePage } from '../actions';
+import ClaimsListItem from '../components/ClaimsListItem';
+import NoClaims from '../components/NoClaims';
+import Pagination from '../../common/components/Pagination';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+import { scrollToTop, setUpPage, setPageFocus } from '../utils/page';
+
+class ClosedClaimsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.changePage = this.changePage.bind(this);
+  }
+  componentDidMount() {
+    if(this.props.allClaims) {
+      this.props.getFilteredClaims(false);
```

Comment:
This is the only difference from the `OpenClaimsPage` component, but not sure best way to reuse the component and change this option